docs/adding-or-modifying-smf-services.adoc

```diff
@@ -0,0 +1,112 @@
+:showtitle:
+:numbered:
+:toc: left
+
+= Adding or modifying SMF services
+
+== SMF services
+
+Go to the `smf/` directory, there you will find all current services or you create a new directory if you're working on a new service.
+
+Each directory can contain a myriad of files depending on the service, but only a `manifest.xml` file is necessary. This file contains the complete set of properties associated with the service instance.
+
+== Packaging
```

Comment:
Thanks for documenting these short-comings. These are default decisions that were made ("rust package" comes from "package name", and "service_name" means "zone name" for zones) but this also serves as good data points for "how the manifest could be made more clear + explicit in the future".